lib/cylc/config.py

```diff
@@ -632,7 +632,8 @@ def __init__(self, suite, fpath, template_vars=None,
                 self.cfg['visualization']['node attributes'].items()):
             for attr in attrs:
                 try:
-                    key, value = re.split('\s*=\s*', attr)
+                    # Check form is 'name = attr'.
+                    _, _ = re.split('\s*=\s*', attr)
```

Comment:
Changes in this module will likely conflict with #2312, as I have worked on them as well.